waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -80,27 +80,13 @@ writeDbSchemaChecksumToFile genProjectRootDirAbs dbSchemaChecksumInProjectRootDi
 
 generatePrismaClient :: Path' Abs (Dir ProjectRootDir) -> IO (Either String ())
 generatePrismaClient genProjectRootDirAbs = do
-  dbSchemaChecksumFileExists <- doesFileExist dbSchemaChecksumFp
```

Comment:
Pulled these checks up a level to be more in line with other ops/generator files.